libs/fsm.c

```diff
@@ -1043,6 +1043,18 @@ int fsm_init(char *module)
 		return 0;
 	}
 
+
+#ifdef HAVE__ICETRANSNOLISTEN
+	_IceTransNoListen("tcp");
+#else
+	fvwm_debug(__func__, "[%s][fsm_init] "
+		   "Security risk: _IceTransNoListen is not available. "
+		   "Cannot stop listening for TCP connections.\n"
+		   "A firewall is strongly recommended. "
+		   "See https://github.com/fvwmorg/fvwm3/issues/1029\n", module_name);
+#endif
+
+
```

Comment:
Absolutely not!

There is no need for this to be logged like this.

First of all, I don't want to log sensationalist messages like this.

Second of all, there's absolutely no security risk at all with FvwmButtons listening on TCP.  FVWM already has an open socket to send commands to (which also includes executing arbitrary commands), so there's no explicit exploit here.

Please remove this entire block, and stop inciting unnecessary FUD.